apache2/msc_pcre.c

```diff
@@ -31,11 +31,7 @@ static apr_status_t msc_pcre_cleanup(msc_regex_t *regex) {
         }
 #else
         if (regex->pe != NULL) {
-#if defined(VERSION_NGINX)
```

Comment:
Seems like you removed here *this* occurrence of `VERSION_NGINX` macro. But I think it would be nice to place it with a separated PR, where we remove the entire Nginx "support" from this module finally. And may be we should announce this elimination before we start it.

Reply:
Indeed. And support for IIS & NETWARE as well I imagine.
Here I removed it because there's no reason to make a difference between Nginx & Apache.